### PR TITLE
Update AppSetting Model

### PR DESCRIPTION
**Pull request by Joeper214**

### CODE REVIEW

File: app/controllers/app_settings_controller.rb [hunk 1]
```diff
@@ -9,14 +9,13 @@
 class AppSettingsController < ApplicationController
   before_action :authenticate_user!, except: [:show, :create, :chef_create]
 
-  before_action except: [:edit_zabbix, :update_zabbix, :chef_server, :chef_keys] do
-    if AppSetting.set?
-      redirect_to root_path
-    end
-  end
+  before_action :installation_complete, except: [:edit_zabbix, :update_zabbix, :chef_server, :chef_keys]
```
Comment (skyarch-networks) [inline app/controllers/app_settings_controller.rb]: __[rubocop]__
* Style/SymbolArray: Use `%i` or `%I` for an array of symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#percent-i)

**Comment by skyarch-networks** [comment]
Click here if you want to create rubocop's auto-correct Pull Request
[Auto Correct by SideCI](https://sideci.com/gh/31754199/issues/pull_request/403/diff/86ebbfb31355a38415ea611e7e30ee77039c0e0c?open_modal=1#rubocop)

See the documentation if you want to get more information about auto-correct.
[Document](https://sideci.com/ja/docs/autofix-rubocop)

**Comment by Joeper214** [comment]
Please review and Merge PR. 🙇  @root-beer

**Comment by root-beer** [comment]
LGTM! Thanks. :bow:

**Comment by Joeper214** [comment]
ありがとう　ございます！。

**Pull request merged**